thoth/build_analysers/preprocessing.py

```diff
@@ -60,7 +60,7 @@ def build_log_to_dependency_table(log: str) -> pd.DataFrame:
     df = pd.io.json.json_normalize(parse_log(log), record_path="result")
 
     if len(df) <= 0:
-        raise ValueError("No packages have been found in the log file.")
```

Comment:
isnt an exception better to handle in the calling code than an empty DF?

Reply:
I think not, I think that raising is unnecessary and it should be up to the user to handle the case when no packages have been found, which is indicated by an empty DataFrame. I think we should not treat it as a critical failure if there are no packages found by the parser (consider for example successful pipenv non-verbose build log, which AFAIK does not have any package names in it and it is perfectly valid.